src/layouts/sidebar-sidecar/components/table-of-contents/types.ts

```diff
@@ -1,7 +1,7 @@
 export interface TableOfContentsHeading {
   title: string
   slug: string
-  level: number
+  level?: number
```

Comment:
The generate-outline function below filters out everything above h2 and in looking at the table of contents component, it doesn't seem to use the level property.